commit e8b73078fd56a4d16ffa9cbb51f43ac75c5f7f1f
Author: nnposter <nnposter@e0a8ed71-7df4-0310-8962-fdc924857419>
Date:   Mon Sep 11 02:33:07 2017 +0000

    Provides more meaningful information about the connecting client when in UDP listening mode. Closes #980

diff --git a/ncat/ncat_listen.c b/ncat/ncat_listen.c
index b047a5866..20b14b7db 100644
--- a/ncat/ncat_listen.c
+++ b/ncat/ncat_listen.c
@@ -898,8 +898,8 @@ static int ncat_listen_dgram(int proto)
             ncat_log_recv(buf, nbytes);
         }
 
-        if (o.debug > 1)
-            logdebug("Valid Connection from %d\n", socket_n);
+        if (o.verbose)
+            loguser("Connection from %s.\n", inet_socktop(&remotess));
 
         conn_inc++;
 
@@ -917,6 +917,7 @@ static int ncat_listen_dgram(int proto)
             struct fdinfo info = { 0 };
 
             info.fd = socket_n;
+            info.remoteaddr = remotess;
             if (o.keepopen)
                 netrun(&info, o.cmdexec);
             else
